src/BlockTypes/ProductCategories.php

```diff
@@ -301,11 +338,15 @@ protected function renderList( $categories, $attributes, $uid, $depth = 0 ) {
 	protected function renderListItems( $categories, $attributes, $uid, $depth = 0 ) {
 		$output = '';
 
+		$link_color_class_and_style = StyleAttributesUtils::get_link_color_class_and_style( $attributes );
+
+		$link_color_style = $link_color_class_and_style['style'];
```

Comment:
This throws a warning on my end: `Warning: Trying to access array offset on value of type null in /wp-content/plugins/woocommerce-gutenberg-products-block/src/BlockTypes/ProductCategories.php on line 343`.